### PR TITLE
Virtual Dedicated Cloud

### PR DESCRIPTION
**Pull request by fiquick**
virtual dedicated cloud

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-416.surge.sh

Updated pages:
- [Connecting to an instance](https://neo4j-docs-aura-416.surge.sh/aura/auradb/getting-started/connect-database)
- [Creating an instance](https://neo4j-docs-aura-416.surge.sh/aura/auradb/getting-started/create-database)
- [Neo4j AuraDB overview](https://neo4j-docs-aura-416.surge.sh/aura/auradb)
- [Monitoring](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/monitoring)
- [Aura with cloud provider marketplaces](https://neo4j-docs-aura-416.surge.sh/aura/platform/cloud-providers)
- [Request and download logs](https://neo4j-docs-aura-416.surge.sh/aura/platform/logging/download-logs)
- [Security log forwarding](https://neo4j-docs-aura-416.surge.sh/aura/platform/logging/log-forwarding)
- [Customer Metrics Integration (CMI)](https://neo4j-docs-aura-416.surge.sh/aura/platform/metrics-integration)
- [Encryption](https://neo4j-docs-aura-416.surge.sh/aura/platform/security/encryption)

**Comment by fiquick** [comment]
[Virtual Dedicated Cloud](https://trello.com/c/j91bMegr/348-virtual-dedicated-cloud)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-416.surge.sh

Updated pages:
- [Connecting to an instance](https://neo4j-docs-aura-416.surge.sh/aura/auradb/getting-started/connect-database)
- [Creating an instance](https://neo4j-docs-aura-416.surge.sh/aura/auradb/getting-started/create-database)
- [Neo4j AuraDB overview](https://neo4j-docs-aura-416.surge.sh/aura/auradb)
- [Advanced metrics](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/advanced-metrics)
- [Backup, export and restore](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Instance actions](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/database-actions)
- [Monitoring](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/monitoring)
- [Authentication](https://neo4j-docs-aura-416.surge.sh/aura/platform/api/authentication)
- [Aura with cloud provider marketplaces](https://neo4j-docs-aura-416.surge.sh/aura/platform/cloud-providers)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-416.surge.sh

Updated pages:
- [Connecting to an instance](https://neo4j-docs-aura-416.surge.sh/aura/auradb/getting-started/connect-database)
- [Creating an instance](https://neo4j-docs-aura-416.surge.sh/aura/auradb/getting-started/create-database)
- [Neo4j AuraDB overview](https://neo4j-docs-aura-416.surge.sh/aura/auradb)
- [Advanced metrics](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/advanced-metrics)
- [Backup, export and restore](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Instance actions](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/database-actions)
- [Monitoring](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/monitoring)
- [Authentication](https://neo4j-docs-aura-416.surge.sh/aura/platform/api/authentication)
- [Aura with cloud provider marketplaces](https://neo4j-docs-aura-416.surge.sh/aura/platform/cloud-providers)

File: modules/ROOT/pages/auradb/getting-started/create-database.adoc [hunk 1]
```diff
@@ -78,14 +78,14 @@ This name can be whatever you like.
...
+[.include-with-AuraDB-Virtual-Dedicated-Cloud]
 =====
 
-To create an *AuraDB Enterprise* instance in Neo4j AuraDB:
+To create an *AuraDB Virtual Dedicated Cloud* instance in Neo4j AuraDB:
 
 . Navigate to the https://console.neo4j.io/?product=aura-db[Neo4j Aura Console] in your browser.
 . Select *New Instance* to open the *Create an instance* page.
-. Set your *Instance size*, the memory, CPU, and storage allocated to the instance. The larger the instance size, the more it costs to run. Please refer to your enterprise contract for pricing.
+. Set your *Instance size*, the memory, CPU, and storage allocated to the instance. The larger the instance size, the more it costs to run. Please refer to your contract for pricing.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/auradb/getting-started/create-database.adoc:88]:
```suggestion
. Set your *Instance size*, the memory, CPU, and storage allocated to the instance. 
Please refer to your contract for pricing.
```

File: modules/ROOT/pages/platform/api/authentication.adoc [hunk 2]
```diff
@@ -8,8 +8,7 @@ The Aura API uses OAuth 2.0 for API authentication.
 
 [NOTE]
 ====
-Enterprise users have unrestricted access to creating API credentials.
-However, users with Free and Professional instances must have entered billing information or be a member of a marketplace tenant before they can create API credentials.
+AuraDB Virtual Dedicated Cloud users, and AuraDS Enterprise users have unrestricted access to creating API credentials. However, users with Free and Professional instances must have entered billing information or be a member of a marketplace tenant before they can create API credentials.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/api/authentication.adoc:11]:
```suggestion
AuraDB Virtual Dedicated Cloud users, and AuraDS Enterprise users have unrestricted access to creating API credentials. 
However, users with Free and Professional instances must have entered billing information or be a member of a marketplace tenant before they can create API credentials.
```

File: modules/ROOT/pages/platform/metrics-integration.adoc [hunk 3]
```diff
@@ -51,6 +52,11 @@ Metrics for a Neo4j Aura instance are only returned if all the following are tru
 * Tenant is of type `enterprise`.
 * The specified instance belongs to the specified tenant.
 
+[CAUTION]
+====
+The legacy term `enterprise` is still used within the codebase and API. However, in the Aura console and documentation, the AuraDB Enterprise tenant type is now known as AuraDB Virtual Dedicated Cloud.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/metrics-integration.adoc:57]:
```suggestion
The legacy term `enterprise` is still used within the codebase and API. 
However, in the Aura console and documentation, the AuraDB Enterprise tenant type is now known as AuraDB Virtual Dedicated Cloud.
```

**Review by AlexicaWright — COMMENTED** [review]
A few suggestions only, looks really good!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-416.surge.sh

Updated pages:
- [Connecting to an instance](https://neo4j-docs-aura-416.surge.sh/aura/auradb/getting-started/connect-database)
- [Creating an instance](https://neo4j-docs-aura-416.surge.sh/aura/auradb/getting-started/create-database)
- [Neo4j AuraDB overview](https://neo4j-docs-aura-416.surge.sh/aura/auradb)
- [Advanced metrics](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/advanced-metrics)
- [Backup, export and restore](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Instance actions](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/database-actions)
- [Monitoring](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/monitoring)
- [Authentication](https://neo4j-docs-aura-416.surge.sh/aura/platform/api/authentication)
- [Aura with cloud provider marketplaces](https://neo4j-docs-aura-416.surge.sh/aura/platform/cloud-providers)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-416.surge.sh

Updated pages:
- [Connecting to an instance](https://neo4j-docs-aura-416.surge.sh/aura/auradb/getting-started/connect-database)
- [Creating an instance](https://neo4j-docs-aura-416.surge.sh/aura/auradb/getting-started/create-database)
- [Neo4j AuraDB overview](https://neo4j-docs-aura-416.surge.sh/aura/auradb)
- [Advanced metrics](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/advanced-metrics)
- [Backup, export and restore](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/backup-restore-export)
- [Instance actions](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/database-actions)
- [Monitoring](https://neo4j-docs-aura-416.surge.sh/aura/auradb/managing-databases/monitoring)
- [Authentication](https://neo4j-docs-aura-416.surge.sh/aura/platform/api/authentication)
- [Aura with cloud provider marketplaces](https://neo4j-docs-aura-416.surge.sh/aura/platform/cloud-providers)

**Pull request merged**